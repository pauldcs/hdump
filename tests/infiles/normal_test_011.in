assets/ls_tail -o 0x3000