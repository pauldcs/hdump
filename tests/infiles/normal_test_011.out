0000+3000  00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00  ................
+
0000+3670  00 00 00 00 00 00 00 00 00 00 00 00 00           .............

